docs/source/content/deep_learning/cnn.rst

```diff
@@ -0,0 +1,237 @@
+#############################
+Convolutional Neural Networks
+#############################
+
+.. contents::
+  :local:
+  :depth: 2
+
+
+********
+Overview
+********
+In the last module, we started our dive into deep learning by talking about
+multi-layer perceptrons. In this module, we will learn about **convolutional
+neural networks** also called **CNNs** or **ConvNets**. CNNs differ from other
+neural networks in that sequential layers are not necessarily fully connected.
+This means that a subset of the input neurons may only feed into a single
+neuron in the next layer. Another interesting feature of CNNs is their inputs.
+With other neural networks we might use vectors as inputs, but with CNNs we
+are typically working with images and other objects with many dimensions.
+*Figure 1* shows some sample images that are each 6 pixels by 6 pixels. The
+first image is colored and has three channels for red, green, and blue values.
+The second image is black-and-white and only has one channel for gray values
+
+.. figure:: _img/Images.png
```

Comment:
add references to images if you used an outside sources, this goes for all images.

Reply:
I made all the images so they should be fine as is. I did add in a reference section like you have in your PR though.